### DISCUSSION TITLE
automatic withoutExceptionHandling when phpunit --debug

### DISCUSSION THREAD

**Discussion by dillingham** [laravel/framework#40761] [Ideas]
would be nice 😄

**Comment by ousid** [comment]
Hey, @dillingham 

Can you explain a bit your idea please?

**Reply by dillingham** [reply]
Sometimes you'll get a "expected 422 but got 500" when doing http tests. Was just thinking it would be cool to just add a --debug flag to the command vs ->withoutExceptionHandling() to the test. Debug mode offers some nice insight as is and it felt like maybe an ok place to add that 

if(in_array('--debug', $_SERVER['argv'], true)) {
     
}

**Comment by crynobone** [comment]
Hey there,

We're closing this issue because it's inactive, already solved, old, or not relevant anymore. Feel free to open up a new issue if you're still experiencing this problem.
